charts/datadog/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: datadog
-version: 3.69.3
+version: 3.69.4
```

Comment:
if it is a "breaking change" for customer event if it is on a beta feature, it could be better to bump the minor version, so user might have chance to read the change log.

Reply:
Fixed, thanks!